tests/hypothesis_strategies/acquisition.py

```diff
@@ -7,19 +7,28 @@
     ProbabilityOfImprovement,
     UpperConfidenceBound,
     qExpectedImprovement,
+    qLogExpectedImprovement,
```

Comment:
Could it be that there are two acqfs missing here?

Reply:
?
I count 12 everywhere including in acqfs

Reply:
oh iw as lokin in init, yes 2 were missing and are added now